app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+	has_many :articles
+	has_secure_password
+
+	validates :email, presence: true, format: {with: /\A[^@\s]+@[^@\s]+\z/, message: "must be a valid Email Address"}
```

Comment:
break line, check once if there are any better validations present for email - https://stackoverflow.com/questions/38611405/email-validation-in-ruby-on-rails